src/common/components/file-issue-details-button.tsx

```diff
@@ -36,6 +37,12 @@ export class FileIssueDetailsButton extends React.Component<FileIssueDetailsButt
         this.state = { showingFileIssueDialog: false, showingHelpText: false };
     }
 
+    private static getDerivedStateFromProps(props: FileIssueDetailsButtonProps): FileIssueDetailsButtonState {
+        if (isString(props.issueTrackerPath) && props.issueTrackerPath.length > 0) {
```

Comment:
Should we trim the `issueTrackerPath` variable before checking for its length to check if some user didn't just put any spaces in?

Reply:
If the user types anything into settings (even a space), they no longer need to be told to go to settings.

Reply:
Discussed offline, Scope of this bug and PR is just to check if there is anything in the url repo to make the settings text disappear! 👍